templates/complete_modular/main.tf

```
@@ -0,0 +1,68 @@
+module "management_resources" {
+  source  = "Azure/alz-management/azurerm"
+  version = "~> 0.1.0"
+
+  automation_account_name      = try(local.management.automation_account_name, "")
+  location                     = try(local.management.location, "")
+  log_analytics_workspace_name = try(local.management.log_analytics_workspace_name, "")
+  resource_group_name          = try(local.management.resource_group_name, "")
```

```suggestion
  resource_group_name          = try(local.management.resource_group_name, "")
    providers = {
    azurerm = azurerm.management
  }
```
Perhaps we could explicitly declare the provider?